vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -134,24 +134,20 @@ $global:map = @{
     "c:\akse-cache\win-k8s\"      = @(
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.12-hotfix.20220922/windowszip/v1.23.12-hotfix.20220922-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.15-hotfix.20230114/windowszip/v1.23.15-hotfix.20230114-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.3-hotfix.20221006/windowszip/v1.24.3-hotfix.20221006-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.6-hotfix.20221006/windowszip/v1.24.6-hotfix.20221006-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.9-hotfix.20230509/windowszip/v1.24.9-hotfix.20230509-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.9-hotfix.20230612/windowszip/v1.24.9-hotfix.20230612-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.10-hotfix.20230509/windowszip/v1.24.10-hotfix.20230509-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.10-hotfix.20230612/windowszip/v1.24.10-hotfix.20230612-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.25.2-hotfix.20221006/windowszip/v1.25.2-hotfix.20221006-1int.zip",
```

Comment:
Does AKS not use v1.25.2-hotfix.20221006?

Reply:
According to the comment "Keep 4 patch versions", since 1.25.11 is added, I remove 1.25.2.

Reply:
Thanks for following the guidance.